ws-client/src/tests.rs

```diff
@@ -116,7 +116,7 @@ async fn response_with_wrong_id() {
 		WebSocketTestServer::with_hardcoded_response("127.0.0.1:0".parse().unwrap(), internal_error(Id::Num(99_u64)))
 			.await;
 	let uri = to_ws_uri_string(server.local_addr());
-	let client = WsClient::new(&uri, WsConfig::default()).await.unwrap();
+	let client = WsClient::new(WsConfig::default_settings_with_url(&uri)).await.unwrap();
```

Comment:
```suggestion
	let client = WsClient::new(WsConfig::default_with_url(&uri)).await.unwrap();
```

Maybe it could be shortened a bit, given it's used often?

Reply:
Reading it I had the same suggestion in mind, partly to make it shorter, partly to not mix language here (config | settings)

Reply:
ok, what about `with_url`?

Reply:
I like `with_url`.